buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -448,10 +448,10 @@
   "quality": "definition",
   "queue": {
     "id": 36,
```

Comment:
The ID's need to be changed too, please see; https://github.com/dotnet/corert/pull/4792/files